Integrations/integration-McAfee-ATD.yml

```diff
@@ -36,687 +37,665 @@ configuration:
-    var PASSWORD = params.password;
-    var UNSECURE = params.unsecure;
-    var PROXY = params.proxy;
-
-    //DICTIONARIES//
-    var LOGINHEADERS = {
-        'Accept': ['application/vnd.ve.v1.0+json'],
-        'Content-Type': ['application/json'],
-        'VE-SDK-API': [Base64.encode(USERNAME + ':' + PASSWORD)]
-    };
-    var HEARTBEATHEADERS = {
-        'Accept': ['application/vnd.ve.v1.0+json'],
-        'Content-Type': ['application/json']
-    };
-
-    //HELPER FUNCTIONS//
-    function stringArrToIntArr(stringArr){
-        var intArr = [];
-        if (stringArr.length > 0){
-            for (index = 0; index < stringArr.length; index++) {
-                if(stringArr[index]){
-                    intArr.push(parseInt(stringArr[index]));
-                }
-            }
-        }
-        else{
-            intArr = [parseInt(stringArr)]
-        }
-        return intArr;
-    }
-
-    function resToMD(body, title){
-        return tblToMd(title, body, Object.keys(body));
-    }
-
-    function castToInt(val){
-        if (typeof val === 'string') {
-            return parseInt(val);
-        } else {
-            return val;
-        }
+    HEARTBEATHEADERS = {
+        'Accept': 'application/vnd.ve.v1.0+json',
+        'Content-Type': 'application/json'
     }
 
-    function getSessionCredentials(){
-        res = sendRequest('/php/session.php','GET', LOGINHEADERS);
-        if (res) {
-            return res.results;
-        }
-    }
-
-    function getHeaders(){
-        var sess = getSessionCredentials();
-        return {'Accept': ['application/vnd.ve.v1.0+json'],
-                'VE-SDK-API':[Base64.encode(sess.session + ':' + sess.userId)]
-        };
-    }
-
-    function sendRequestRaw(uri, method, headers, body){
-        var res = http(
-                BASE+uri,
-                {
-                    Method: method,
-                    Headers: headers,
-                    Body: body
-                },
-                UNSECURE,
-                PROXY
-            );
-        if (res.StatusCode < 200 || res.StatusCode >= 300) {
-            throw 'Failed to use uri ' + BASE+uri + ', request status code: ' + res.StatusCode + ' and Body: ' + res.Body + '.';
+    ''' HELPERS '''
+    def stringArrToIntArr(stringArr):
+        ''' Converts a string array to int array '''
+        intArr = []
+        if isinstance(stringArr, list):
+            for index in range(len(stringArr)):
+                if stringArr[index]:
+                    intArr.append(int(stringArr[index]))
+        else:
+            intArr = [int(stringArr)]
+        return intArr
+
+    def castToInt(val):
+        ''' Casts a string to int '''
+        if isinstance(val, str):
+            return int(val)
+        else:
+            return val
+
+    def getSessionCredentials():
+        res = sendRequest('php/session.php','get', LOGINHEADERS)
+        if (res):
+            return res['results']
+
+    def getHeaders():
```

Comment:
create the headers in the beginning and then use them

Reply:
done